.teamcity/Gradle_Check_dsl.iml

```diff
@@ -32,9 +32,9 @@
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" name="Maven: org.jetbrains.teamcity:configs-dsl-converters:2017.1.3" level="project" />
```

Comment:
Strange that `configs-dsl-converters` is still version `2017.1.3`.

Reply:
Fixed.

Reply:
How? Did you manually edit the file?

Reply:
No, re-imported the project with IDEA.